impl/src/main/java/dao/QuestionDao.java

```diff
@@ -24,12 +26,13 @@
         "FROM " +
             "question " +
         "INNER JOIN " +
-            "\"user\" on \"user\".id = question.user_id WHERE question.user_id=:userId " +
-        "LIMIT " +
-        ":limit")
-    Observable<Question> getQuestions(long userId, Integer limit);
+            "\"user\" on \"user\".id = question.user_id WHERE question.user_id=:userId ",
+        defaultLimit = 10,
```

Comment:
Finally my turn to suggest it, could these be made constants instead? :)

Reply:
I guess you and @magnusp need to battle it out then. Leaving as is for now.